### PR TITLE
save img for debug

### PR DESCRIPTION
**Pull request by MateoLostanlen**
It can be useful to save the images on the pi for debug purposes. This PR adds this option, you just have to add a variable save_img_debug to config_data.json:
```

{
    "detection_threshold": 0.25,
    "api_url": "https://api.pyronear.org",
    "save_evry_n_frame": 50,
    "loop_time": 7.5,
    "latitude": 48.8673439026,
    "longitude": 2.3841824532,
    "model_weights": "data/yolov5s_v001.onnx",
    "save_img_debug": True,
}

```

### CODE REVIEW

File: runner/src/runner.py [hunk 1]
```diff
@@ -71,6 +72,12 @@ def capture(ip, CAM_USER, CAM_PWD):
             img = capture(ip, CAM_USER, CAM_PWD)
             pred = engine.predict(img, ip)
 
+            # Save img
+            if save_img_debug:
+                file = 'data/' + ip + '/' + time.strftime("%Y%m%d-%H%M%S") + '.jpg'
```
[the remark this answers is not in the document]
Comment (MateoLostanlen) [reply runner/src/runner.py:77]: yes

File: runner/src/runner.py [hunk 2]
```diff
@@ -71,6 +72,12 @@ def capture(ip, CAM_USER, CAM_PWD):
...
+                os.makedirs(os.path.split(file)[0], exist_ok=True)
+                img.save('data/' + ip + '/.jpg')
```
[the remark this answers is not in the document]
Comment (MateoLostanlen) [reply runner/src/runner.py]: thnaks

File: runner/src/runner.py [hunk 3]
```diff
@@ -71,6 +72,12 @@ def capture(ip, CAM_USER, CAM_PWD):
             img = capture(ip, CAM_USER, CAM_PWD)
             pred = engine.predict(img, ip)
 
+            # Save img
+            if save_img_debug:
+                file = 'data/' + ip + '/' + time.strftime("%Y%m%d-%H%M%S") + '.jpg'
+                os.makedirs(os.path.split(file)[0], exist_ok=True)
```
[the remark this answers is not in the document]
Comment (frgfm) [reply runner/src/runner.py:78]: Oh my bad, I thought only `pathlib.Path.makedir` was allowing parent folder creation :+1:

**Review by frgfm — APPROVED** [review]
Looks good, thanks!

**Pull request merged**